cockroachcloud/migrations-page.md

```diff
@@ -0,0 +1,159 @@
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Migration Name | The filename of the `.sql` file that was [uploaded](#upload-a-sql-dump).                                                                                                                       |
+| Status         | The status of the migration: `READY FOR REVIEW`, `READY TO FINALIZE`, or `FINALIZED`. Migrations with `READY TO FINALIZE` status can be [finalized](#finalize-the-schema).                    |
+| Date Imported  | The timestamp when the SQL dump was uploaded.                                                                                                                                                |
+| Last Updated   | The timestamp when the [SQL statements](#statements-list) were updated.                                                                                                                      |
+| Errors         | The number of SQL errors preventing a migration from attaining `READY TO FINALIZE` status. |
+
+To view the [**Summary Report**](#summary-report) or [**Statements** list](#statements-list) for a migration, click the migration name.
+
+## Summary Report
+
+The **Summary Report** displays the results of the schema analysis:
+
+- The number of **statements total** that were analyzed in the `.sql` file that you [uploaded](#upload-a-sql-dump).
+- The number of **errors** in SQL statements that are blocking [finalization](#finalize-the-schema). Errors are further categorized on the [**Statement Status**](#statement-status) graph.
+- The number of **incidental errors** in SQL statements that are caused by errors in other SQL statements.
+- The number of **suggestions** that were made regarding [differences from other databases](../{{version_prefix}}migration-overview.html#differences-from-other-databases).
+
+To resolve errors and review suggestions, click **View Statements** or the **Statements** tab to open the [**Statements** list](#statements-list).
+
+To [finalize the schema](#finalize-the-schema) and create a new database for migration, click **Finalize Schema**. The number of **errors** must be zero.
+
+### Statement Status
+
+The **Statement Status** graph displays the number of successful statements (green), the number of errors (red), and the number of incidental errors (orange):
+
+- **OK** represents a successful statement.
+- **Unimplemented Feature** represents a statement that uses an [unimplemented feature](../{{version_prefix}}migration-overview.html#unimplemented-features).
+- **Statement Error** represents a statement that failed for a reason unrelated to an unimplemented feature or missing user.
```

Comment:
Could just be me, but parsing "...failed for a reason unrelated to an unimplemented feature or missing user." is quite hard, because of the double "un" prefix. I re-read and understood the sentence fine. In my head I wanted to say, "for a reason that is not related to an unimplemented feature..." But that's kind of weird.

Would it still make the same sense if you switched "unimplemented feature" and "missing user"?

So, "failed for a reason unrelated to a missing user or an unimplemented feature."

Reply:
Switched and also rephrased to "other than"! I think this helps.